docs/security-privacy-self-review.md

```diff
@@ -0,0 +1,98 @@
+
+
+Responses to the [Self-Review Questionnaire: Security and Privacy](https://w3ctag.github.io/security-questionnaire/) for the [Raw Sockets API](https://github.com/WICG/raw-sockets/)
+
+
+
+
+## 2.1 What information might this feature expose to Web sites or other parties, and for what purposes is that exposure necessary?
+
+When the user agrees to establish a socket connection with an information system, and the connection succeeds, that system's IP address and port, and any information it sends to the user agent, becomes available to the site.
```

Comment:
Information is shared in both directions, about the user's system and the destination system, right?
Maybe add in that the destination system also receives the user's IP address and port, and any information the site sends?

Reply:
done